packages/office-ui-fabric-react/src/components/Modal/Modal.types.ts

```diff
@@ -55,7 +55,7 @@ export interface IModalProps extends React.ClassAttributes<ModalBase>, IWithResp
   /**
    * Props to be passed through to Layer
    */
-  layerProps?: ILayerProps;
+  layer?: ILayerProps;
```

Comment:
Why are you changing the name of this prop, seems completely unrelated to what this PR is trying to fix (and it's a breaking change)

Reply:
@JasonGore asked the following in https://github.com/OfficeDev/office-ui-fabric-react/pull/8273#discussion_r266679679:

> I should have mentioned this earlier but can you call this prop `layer` instead of `layerProps`? It will line up with the `layer` style section naming and with Slots functionality we're introducting with Fabric 7, and will help ensure this component is backwards compatible. Thanks! 

@aftab-hassan if you rename the prop, you should keep `layerProps` and mark it as deprecated (use @deprecated), and then make sure that both the new `layer` and the old `layerProps` work in the same way. This is to maintain backwards compatibility.

Reply:
Here's an example of how we do this for `collapsed` and `isCollapsed` in SearchBox:

https://github.com/OfficeDev/office-ui-fabric-react/blob/78a6157e3caa57db9634d0b11d245b3187f573e6/packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.types.ts#L26-L35

https://github.com/OfficeDev/office-ui-fabric-react/blob/78a6157e3caa57db9634d0b11d245b3187f573e6/packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx#L29-L32

https://github.com/OfficeDev/office-ui-fabric-react/blob/78a6157e3caa57db9634d0b11d245b3187f573e6/packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx#L69

Reply:
I'm sorry, @jspurlin is completely right. For some reason I thought `layerProps` was a new prop added in this PR. Don't worry about renaming or deprecation as part of this PR. I'm not sure how I misintepreted that. 😕

Reply:
Thanks for the sample @natalieethell 

I'll revert the naming @JasonGore